src/StreamJsonRpc/MessageFormatterHelper.cs

```diff
@@ -0,0 +1,24 @@
+﻿
+namespace StreamJsonRpc
+{
+    using System;
+    using System.Linq;
+    using System.Reflection;
+
+    public class MessageFormatterHelper
+    {
+        public static Type FindIProgressOfT(Type objectType)
```

Comment:
>public [](start = 8, length = 6)

If this will be public, we'll definitely want it adequately documented via xml doc comments.